packages/gitbook/src/lib/v1.ts

```diff
@@ -48,16 +48,21 @@ export async function getV1BaseContext(): Promise<GitBookBaseContext> {
     const host = await getHost();
     const basePath = await getBasePath();
     const siteBasePath = await getSiteBasePath();
+    const protocol = await getProtocol();
 
     const linker = createLinker({
+        protocol,
         host,
         spaceBasePath: basePath,
         siteBasePath: siteBasePath,
     });
 
     // On V1, we use hard-navigation between different spaces because of layout issues
-    linker.toLinkForContent = (url) => {
-        return url;
+    linker.toLinkForContent = (rawURL) => {
```

Comment:
Why do we change this?

Reply:
I wanted to fix the fact that in proxy mode URL are wrong, but it seems to break things.

Reply:
Proxy mode is only supported in v2, it was on purpose

Reply:
I am not talking about this proxy mode. In dev the URL are wrong for sections, sitemap and other places. So I was trying to fix that.

Reply:
I removed it from this PR.